stubs/openpyxl/@tests/test_cases/check_nested_descriptors.py

```diff
@@ -32,7 +32,7 @@
 
 
 class WithDescriptors(Serialisable):
-    descriptor = Nested[str]()
+    descriptor = Nested(expected_type=str)
```

Comment:
This can't really work as a type annotation, can it? (Same for the existing attributes below.)

Reply:
It"s not a type annotation, it's an actual assignement.